influence_benchmark/config/experiment_configs/KTO_therapist_final.yaml

```diff
@@ -0,0 +1,48 @@
+# Run settings
+run_name: "therapist-final-reward"
+devices: [0, 1, 2, 3, 4, 5, 6, 7]  # GPUs used for generating trajectories
+log_to_wandb: true
+
+# Specify settings for generating trajectories
+env_name: "therapist-final"
+max_turns: 5
+num_envs_per_device: 12 # number of environment slots to be filled with env-subenv-initialstate combinations. 8 is roughly max for 5-turn convos
+max_subenvs_per_env: 2
+final_reward: true
+
+# Trajectory generation settings
+num_gen_trajs_per_initial_state: 16
+top_n_trajs_per_initial_state: 1
+iterations: 10
+
+# Model settings
+agent_model_name: "meta-llama/Meta-Llama-3-8B-Instruct"
+env_model_name: "meta-llama/Meta-Llama-3-8B-Instruct"
+
+# Training settings
+per_device_train_batch_size: 1
+num_train_epochs: 1
+gradient_accumulation_steps: 32
+gradient_checkpointing: true
+learning_rate: 1.0e-4
+report_to: "none"
+optim: "adamw_torch"
+max_seq_length: 4096
+lr_scheduler_type: "constant"
+logging_steps: 1
+
+# LoRA hyperparameters
+lora_r: 8
+lora_alpha: 32
+lora_dropout: 0.1
+
+# KTO specific settings
+beta: 0.1
+desirable_weight: 1.0
+undesirable_weight: 1.0
```

Comment:
desirable_weight and undesirable_weight should no longer be specified as they are calculated such that the ratio is 1.05. we should probably add a parameter which is the ratio

Reply:
I'm a bit confused about this – even if the ratio is 1.05, the scale of desirable and undesirable could be quite different?

Reply:
I guess are you saying we should just use `num_positives / num_negatives` for the undesirable weight and then use the ratio to figure out the desirable weight?

Reply:
Yeah, only the ratio between them actually matters